### PR TITLE
Added how to use Film Discovery

### PR DESCRIPTION
**Pull request by alexflores37**
added gif of how to use film discovery

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ Film Discovery- An application to be able to access Your favorite movie's Popula
 ## Description
 We had created this application to give users a way to find out the popularity of a movie of their choice via the amount of times the wikipedia page for that specified movie was viewed by a human. Also in this application the user will be able to view the movie's Popularity, Release Date, Box Office, Run Time, Genre, and display the Movie's Poster. One thing this application helped us learn the nuances of the Fetch Function. 
 ## Screenshot
+![How to Use Film Discovery](./assets/images/How%20To%20use%20Film%20Discovery.gif)
```
Comment (ameridacas) [inline README.md:7]: nice a gif of our page working

File: index.html [hunk 2]
```diff
@@ -44,7 +44,8 @@ <h1 class="text-white text-2xl" style="text-shadow: 0 0 10px red;"></h1>
                         <input id="rating" type="text" value="displaySearch" class="border-2 w-auto mt-2 rounded-md justify-items-end ">
                         <label for="releaseDate" class="block text-lg text-white mb-10px ml-2 justify-items-end">ReleaseDate</label>
                         <input id="releaseDate" type="text" value="displaySearch" class="border-2 w-auto mt-2 p-10px rounded-md justify-items-end">-->
-                    <div>
+                    <ul>
```
Comment (ameridacas) [inline index.html:47]: we don't need  a fun fact title over the api info displayed

Comment (alexflores37) [reply index.html:47]: OK!

**Review by ameridacas — COMMENTED** [review]
I made comments on the code itself. You need to remove the fun fact section and ul element

File: README.md (hunk 1 above)
Comment (alexflores37) [reply README.md:7]: Perfect!

**Review by ameridacas — APPROVED** [review]
Nice job!

**Pull request merged**